### PR TITLE
up page14

### PR DESCRIPTION
**Pull request by binhan612**
https://edu-redmine.sun-asterisk.vn/issues/36929
Screenshot_1
Screenshot_2
Screenshot_3
Screenshot_4

### CODE REVIEW

**Comment by binhan612** [comment]
ready

**Comment by binhan612** [comment]
ready

**Comment by binhan612** [comment]
ready

File: pages/contact-page.pug [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+				form(action="/action_page.php")
+					+input('name', 'Your name', 'text', 'name', '', '')
+					+input('email', 'Your emial', 'email', 'email', '', '')
+					+input('subject', 'Your subject', 'text', 'subject', '', '')
+					div.key-in
+						label(for='message') Your message
+						textarea(id="message" name="message" rows="4")
+					+input('submit', '', 'submit', 'submit', 'Gửi', '')
+			div.info
+				img(src='assets/image/Screenshot_4.png')
```
Comment (vietnq-1097) [inline pages/contact-page.pug]: Chị bổ sung thuộc tính alt nhé

**Comment by binhan612** [comment]
ready

**Pull request merged**